.circleci/config.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     docker:
       # Our policy for updating this rust version is at https://github.com/mozilla/uniffi-rs/tree/main/docs/policies/rust-versions.md
       # See also rust-toolchain.toml in the root of this repo, which is used to ensure the local version of rust matches.
-      - image: circleci/rust:1.47.0
+      - image: circleci/rust:1.52.1
```

Comment:
Actually, I wonder if it would be simpler for us to just use the `uniffi-ci` docker image for running these tests as well, then we don't have to specify an explicit version number here. Not sure how this would affect our CircleCI costs or anything like that, but it'd be simpler.